app/src/main/res/layout/app_version_fragment.xml

```diff
@@ -0,0 +1,74 @@
+    <variable
+      name="viewModel"
+      type="org.oppia.app.administratorcontrols.appversion.AppVersionViewModel" />
+  </data>
+  <FrameLayout
+    android:layout_width="match_parent"
+    android:layout_height="match_parent">
+    <androidx.constraintlayout.widget.ConstraintLayout
+      android:layout_width="match_parent"
+      android:layout_height="match_parent">
+
+      <TextView
+        android:id="@+id/app_version_text_view"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:background="@color/white"
+        android:fontFamily="sans-serif"
+        android:padding="16dp"
+        android:text="@{@string/app_version_name(viewModel.versionName)}"
+        android:textColor="@color/oppiaPrimaryTextDark"
+        android:textSize="16sp"
+        app:layout_constraintBottom_toTopOf="@id/view_divider"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent" />
+
+      <View
+        android:id="@+id/view_divider"
+        android:layout_width="match_parent"
+        android:layout_height="2dp"
+        android:background="@color/oppiaGreyBorder"
+        app:layout_constraintBottom_toTopOf="@id/view_divider"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@id/app_version_text_view" />
+
+      <ImageView
+        android:id="@+id/app_info_image_view"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="24dp"
+        android:src="@drawable/ic_info_icon_gray_24dp"
+        app:layout_constraintEnd_toStartOf="@id/app_last_update_date_text_view"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="@id/app_last_update_date_text_view" />
+
+      <TextView
+        android:id="@+id/app_last_update_date_text_view"
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="12dp"
+        android:layout_marginTop="28dp"
+        android:layout_marginEnd="28dp"
+        android:fontFamily="sans-serif"
+        android:text="@{@string/app_last_update_date(viewModel.lastUpdateDate)}"
+        android:textColor="@color/oppiaSecondaryText"
+        android:textSize="12sp"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintHorizontal_weight="1"
+        app:layout_constraintStart_toEndOf="@id/app_info_image_view"
+        app:layout_constraintTop_toBottomOf="@id/view_divider" />
+    </androidx.constraintlayout.widget.ConstraintLayout>
+    <View
+      android:id="@+id/app_version_shadow_view"
+      android:layout_width="match_parent"
+      android:layout_height="6dp"
+      android:layout_gravity="top"
+      android:background="@drawable/toolbar_drop_shadow" />
+  </FrameLayout>
```

Comment:
Press `Control + Alt + L` - reformatting shortcut to fix the formatting.

Reply:
Done